Packages/com.unity.inputsystem/InputSystem/Plugins/PlayerInput/InputValue.cs

```diff
@@ -35,6 +35,43 @@ public object Get()
         }
 
         ////TODO: add automatic conversions
+        /// <summary>
+        /// Read the value of the action.
```

Comment:
Should we add a "current" in there to emphasise?

Reply:
Good idea - doing it